app/helpers/application_helper.rb

```diff
@@ -72,4 +72,29 @@ def colored_span(color, content)
       content
     end
   end
+
+ def easter
```

Comment:
Assignment Branch Condition size for easter is too high. [63.33/15]<br>Inconsistent indentation detected.